### PR TITLE
Fix map loading in Tree display mode

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

File: src/components/horizontal-toolbar.js [hunk 1]
```diff
@@ -179,9 +182,6 @@ export function HorizontalToolbar({ setStudyDisplayMode, studyDisplayMode }) {
     );
 }
 
-HorizontalToolbar.propTypes = {
-    setStudyDisplayMode: PropTypes.func,
-    studyDisplayMode: PropTypes.string,
-};
+HorizontalToolbar.propTypes = {};
```
Comment (etiennehomer) [inline src/components/horizontal-toolbar.js:185]: We keep the empty props ?

**Review by etiennehomer — APPROVED** [review]

**Review by etiennehomer — CHANGES_REQUESTED** [review]
We get Lines and Substations when we change the node in Tree display mode. Maybe we could avoid those requests ?

File: src/components/horizontal-toolbar.js (hunk 1 above)
Comment (sBouzols) [reply src/components/horizontal-toolbar.js]: removed

**Review by etiennehomer — APPROVED** [review]

File: src/components/app-top-bar.js [hunk 2]
```diff
@@ -294,14 +311,11 @@ const AppTopBar = ({
...
-                searchTermDisabled={!isNodeBuilt(currentNode)}
-                initialSearchTerm={
-                    !isNodeBuilt(currentNode)
-                        ? intl.formatMessage({
-                              id: 'InvalidNode',
-                          })
-                        : ''
+                searchTermDisabled={
+                    !isNodeBuilt(currentNode) ||
+                    studyDisplayMode === STUDY_DISPLAY_MODE.TREE
```
Comment (FranckLecuyer) [inline src/components/app-top-bar.js:316]: The test could be : getDisableReason() !== ''   ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/app-top-bar.js (hunk 2 above)
Comment (sBouzols) [reply src/components/app-top-bar.js]: :+1: done

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**